src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs

```diff
@@ -128,6 +128,10 @@ public async Task<IActionResult> CreatePost(TemplateViewModel model, string subm
                 {
                     ModelState.AddModelError(nameof(TemplateViewModel.Name), S["The name is mandatory."]);
                 }
+                else if (String.IsNullOrWhiteSpace(model.Content))
+                {
+                    ModelState.AddModelError(nameof(TemplateViewModel.Content), S["The content is mandatory."]);
```

Comment:
you might as well add this validation to the `Edit` action as well

Reply:
> I made a few changes in your PR (to be applied on Edit too).

I though Antoine add it ;)